webui/react/vite-plugin-branding.ts

```diff
@@ -0,0 +1,21 @@
+import type { Plugin } from 'vite';
+
+export const brandHtml = (): Plugin => {
+  return {
+    name: 'brandHtml',
+    async transformIndexHtml(html: string) {
+      if (process.env.DET_BUILD_EE === 'true') {
+        const cheerio = await import('cheerio');
+        const $ = cheerio.load(html);
+        $('meta').each(function () {
```

Comment:
should be able to just use `'meta[name="description"]'` to skip over non relevant meta tags